packages/storage/amplify_storage_s3/example/integration_test/main_test.dart

```diff
@@ -0,0 +1,623 @@
+              (WidgetTester tester) async {
+            final operation = Amplify.Storage.getUrl(
+              key: testObjectKey2,
+              options: S3GetUrlOptions.forIdentity(
+                user1IdentityId,
+                checkObjectExistence: true,
+              ),
+            );
+
+            await expectLater(operation.result, completes);
+          });
+
+          testWidgets(
+              'get url of object (belongs to other user) with access level private for the currently signed user throws exception',
+              (WidgetTester tester) async {
+            final operation = Amplify.Storage.getUrl(
+              key: testObjectKey3,
+              options: const S3GetUrlOptions(
+                accessLevel: StorageAccessLevel.private,
+                checkObjectExistence: true,
+              ),
+            );
+
+            await expectLater(operation.result, throwsA(isA<S3Exception>()));
+          });
+
+          testWidgets(
+              'download data of object with access level protected and a target identity id for the currently signed user',
+              (WidgetTester tester) async {
+            final result = await Amplify.Storage.downloadData(
+              key: testObjectKey2,
+              options: S3DownloadDataOptions.forIdentity(
+                user1IdentityId,
+                getProperties: true,
+              ),
+            ).result;
+
+            expect(result.downloadedItem.eTag, object2Etag);
+          });
+
+          testWidgets(
+              'download data of object (belongs to other user) with access level private for the currently signed user',
+              (WidgetTester tester) async {
+            final operation = Amplify.Storage.downloadData(
+              key: testObjectKey3,
+              options: const S3DownloadDataOptions(
+                accessLevel: StorageAccessLevel.private,
+                getProperties: true,
+              ),
+            );
+
+            await expectLater(operation.result, throwsA(isA<S3Exception>()));
+          });
+
+          testWidgets(
+              'move object with access level guest for the currently signed in user',
+              (WidgetTester tester) async {
+            final result = await Amplify.Storage.move(
+              source: S3ItemWithAccessLevel(
+                storageItem: S3Item(key: testObjectKey1),
+                accessLevel: StorageAccessLevel.guest,
+              ),
+              destination: S3ItemWithAccessLevel(
+                storageItem: S3Item(key: testObject1MoveKey),
+                accessLevel: StorageAccessLevel.private,
+              ),
+              options: const S3MoveOptions(getProperties: true),
+            ).result;
+
+            expect(result.movedItem.eTag, isNotEmpty);
+          });
+
+          testWidgets(
+              'copy object (belongs to other user) with access level protected for the currently signed in user',
```

Comment:
Can you break up lines longer than 80 chars into multiline strings?

```suggestion
              'copy object (belongs to other user) with access level protected '
              'for the currently signed in user',
```